src/status_im2/contexts/chat/messages/list/view.cljs

```diff
@@ -211,7 +204,7 @@
                                                      [30 125]
                                                      [14 0]
                                                      header-extrapolation-option)]
-    [rn/view (add-inverted-y-android {:flex 1})
+    [rn/view {:flex 1}
```

Comment:
wow that really old, and i believe they just released the fix, nice

Reply:
yeah @qoqobolo discovered this in ISSUE 3 :D